src/_help/enhance-documentation-content/markdown-support.md

```diff
@@ -52,32 +56,32 @@ will render:
 
 ## Information call-outs
 
-Bump support information call-outs (of type `info`, `warn`, `success` or `error`) with the quote markdown syntax (lines starting with `> ` ) if the first line contains one of the call-out types.
+Bump.sh supports information call-outs (of type `info`, `warn`, `success` or `error`) with the quote markdown syntax (lines starting with `> ` ) if the first line contains one of the call-out types.
 
 E.g.
 
 ```undefined
 > info
-> this is an important information to **standout**
+> this is an important information to **standout**.
 ```
 
 will render:
 
-> this is an important information to **standout**
+> this is an important information to **standout**.
 {: .info}
 
 ## Images
 
-Use the following syntax to add images in your markdown
+Use the following syntax to add images in your markdown:
 ```
 ![Alt text](/path/to/image.jpg "Image title")
 ```
 
-Note that even if it's a best practice to always add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, the parameters `Alt text` and `"Image title"` are optional.
+We strongly recommend to add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, even if the parameters `Alt text` and `"Image title"` are optional.
```

Comment:
I would even say something among those lines: 
```suggestion
You should add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) when introducing an image, to make it accessible to everyone.
```
or 
```suggestion
Don't forget to add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, to make it accessible to everyone.
```
Reminding people it's optional might encourage them not to put them but they really really should have alts for every image.